resolwe_bio/processes/import_data/sra_file.py

```diff
@@ -167,6 +173,10 @@ class Input:
         class Advanced:
             """Advanced options."""
 
+            prefetch = BooleanField(label="Prefetch SRA file", default=True)
+            max_size_prefetch = StringField(
+                label="Maximum file size to download in KB", default="20G"
```

Comment:
It is not entirely clear that unit prefixes can be used. Perhaps it could be explained in the description.

Reply:
I have added a description as suggested.